js/index.js

```diff
@@ -37,6 +37,152 @@ const siteContent = {
+
+
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+// END navBar
+
+
+// cta build
+let ctaH1 = document.querySelector('.cta-text h1');
+ctaH1.innerHTML = 'DOM <br> IS <br> Awesome';
+
+let ctaButton = document.querySelector('.cta-text button');
+ctaButton.textContent = "Do NOT Click";
+ctaButton.addEventListener("click", function(){ 
+  alert('You now have a virus. Deleting hard drive. Follow the rules next time.');
+  document.querySelector('.container').style.visibility = "hidden";
+  document.querySelector('body').style.backgroundColor = "red";
+});
+
+let ctaImg = document.getElementById("cta-img");
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"]);
+// END cta build
+
+
+// mainContent build
+let featuresH4 = document.querySelectorAll('.top-content .text-content h4')[0];
+featuresH4.textContent = siteContent['main-content']['features-h4'];
+
+let featuresContent = document.querySelectorAll('.top-content .text-content p')[0];
+featuresContent.textContent = siteContent['main-content']['features-content'];
+
+let aboutH4 = document.querySelectorAll('.top-content .text-content h4')[1];
+aboutH4.textContent = siteContent['main-content']['about-h4'];
+
+let aboutContent = document.querySelectorAll('.top-content .text-content p')[1];
+aboutContent.textContent = siteContent['main-content']['about-content'];
+
+let middleImg = document.getElementById('middle-img');
+middleImg.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+let servicesH4 = document.querySelectorAll('.bottom-content .text-content h4')[0];
+servicesH4.textContent = siteContent['main-content']['services-h4'];
+
+let servicesContent = document.querySelectorAll('.bottom-content .text-content p')[0];
+servicesContent.textContent = siteContent['main-content']['services-content'];
+
+let productH4 = document.querySelectorAll('.bottom-content .text-content h4')[1];
+productH4.textContent = siteContent['main-content']['product-h4'];
+
+let productContent = document.querySelectorAll('.bottom-content .text-content p')[1];
+productContent.textContent = siteContent['main-content']['product-content'];
+
+let visionH4 = document.querySelectorAll('.bottom-content .text-content h4')[2];
+visionH4.textContent = siteContent['main-content']['vision-h4'];
+
+let visionContent = document.querySelectorAll('.bottom-content .text-content p')[2];
+visionContent.textContent = siteContent['main-content']['vision-content'];
+//END mainContent build
+
+//LOOK AT THIS TRASH!!!
+
+// let featuresH4 = document.createElement('h4')
+// featuresH4.textContent = "Features"
+
+// let featuresContent = document.createElement('p')
+// featuresContent.textContent = "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
```

Comment:
you shouldn't have to add your content from your JSON object, you want to keep that content where it is at and find other solutions to inject it into the DOM